packages/noir-ethereum-api/src/ethereum/recordingClient.ts

```diff
@@ -49,6 +51,10 @@ function createLoggingProxy<Method, Target extends { [key: string]: Method }>(
   return new Proxy(target, handler) as Target & GetCalls;
 }
 
+export async function saveCallsToFile(client: RecordingClient, filePath: string): Promise<void> {
+  return fs.writeFile(filePath, stringify(await client.getCalls()));
+}
```

Comment:
```ts
function writeObject(object: object, path: string) {
      return fs.writeFile(filePath, stringify(object));
}
```

So instead of:
```ts
await saveCallsToFile(client, tempFilePath);
```
This:
```ts
await saveObject(client, await client.getCalls());
```

Reply:
And move to `utils/fs.ts`